### ISSUE TITLE
Sops yaml reference documentation

### ISSUE THREAD

**Issue by isavl** [mozilla/sops#998]
Hello there.

Where I can find reference documentation for sops.yaml configuration file?

**Comment by rjhenry** [comment]
I don't think there is any authoritative documentation, as yet. The [`config.go`](https://github.com/mozilla/sops/blob/master/config/config.go) file shows the structures expected (e.g. a creation rule is defined [here](https://github.com/mozilla/sops/blob/master/config/config.go#L110)).

It's not as convenient as a full documentation page, but it will probably help with structuring a `.sops.yaml` file!

**Comment by austinbutler** [comment]
Even just having an example YAML file with all the keys in it would help.

**Comment by almereyda** [comment]
Yes, this is very much needed. And if only in an explanatory manner in the README. It's used there a few times, but one cannot find a human-language description about how it works, and how it is supposed to be used.

**Comment by chriscarpenter12** [comment]
I was [told to add the age keys to .sops.yaml](https://github.com/getsops/sops/issues/1078#issuecomment-2099711116) and couldn't find an example how. I too was looking for an example or docs.

**Comment by bow** [comment]
I know there are several in-memory examples in the [config_test.go](https://github.com/getsops/sops/blob/main/config/config_test.go) file. Still, I (and many others I'm sure) would really appreciate a proper documentation page explaining the config file.

Searching for it was quite frustrating ~ reading a proper documentation would have saved time.
